ydb/library/yql/core/cbo/cbo_optimizer.h

```diff
@@ -1,75 +1,100 @@
 #pragma once
 
 #include <util/generic/vector.h>
+#include <util/generic/string.h>
+#include <ydb/library/yql/core/yql_statistics.h>
+#include <ydb/library/yql/core/yql_cost_function.h>
 
 #include <unordered_map>
+#include <memory>
 #include <map>
+#include <sstream>
+
 
 namespace NYql {
 
-struct IOptimizer {
```

Comment:
Don't delete it. Rename it to IOptimizerOld. Parts of this code will be used internally in PG optimizer.